### ISSUE TITLE
Background Image of DMG is not fit to screen?

### ISSUE THREAD

**Issue by bradennapier** [electron-userland/electron-builder#1995]
![image](https://user-images.githubusercontent.com/15365418/29738687-c1a28826-89de-11e7-9809-bab7088ecc91.png)

When setting a background image - it is setting the window size properly to the image resolution but the image itself is minimzed to just a corner?

**Comment by develar** [comment]
Because you must provide x2 image. Please https://github.com/electron-userland/electron-builder/issues/1506#issuecomment-297634178 

We should document it.

Duplicates #1506

**Comment by afreeland** [comment]
Wanted to note that I experienced this issue too while creating my `background.tiff` and `background@2x.tiff` files in Photoshop with a `width = 540` and `height = 380`.   For me the document started out at a resolution of `144` and then I tried `300` but it kept getting worse.  Finally, I remembered that it typically used to default to `72`, so I tried that not expecting it to work and BAM...matched right up perfectly.   Wanted to post this in case anyone stumbled across this same issue where resolution may be to blame and not at all with the code.

![image](https://user-images.githubusercontent.com/3135030/39943766-14b90a52-5532-11e8-8c36-363daff0e2ed.png)

And the final product looks like:
![image](https://user-images.githubusercontent.com/3135030/39943788-23e2808a-5532-11e8-84e6-4b1aa677fe55.png)
